packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -327,12 +331,22 @@ const MainScreen = (props: MainScreenProps): JSX.Element => {
         />
       );
       break;
+    case 'leaving':
+      pageElement = (
+        <NoticePage
```

Comment:
Any concerns we don't show mic and camera status on this page so users may think they're no longer transmitting these? (I can't recall if they proved media could get through or not in this state)

Reply:
Because we can no longer come back to the calling status for any chance - so stop showing the bar would be the best choice according to what Alex P designed

Disconnecting might still transmit video and audio for some milliseconds, but it has almost 100% chance to stop them correctly cause it is just a local operation, the only thing could be stuck is network related operation - saying that your stop all transmission but your tile is still on others' video gallery